webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -68,58 +68,7 @@ msgstr ""
 "wenn Sie mit Ihrer Steuererklärung fortfahren möchten. Wenn Sie Ihren "
 "Freischaltcode stornieren möchten, müssen Sie sich zuerst abmelden."
 
-#: app/routes.py:187 app/routes.py:295
-msgid "404.header-title"
-msgstr "Fehler 404 - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:207 app/templates/basis/base.html:36
-msgid "page.title"
-msgstr "Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:214
-msgid "howitworks.header-title"
-msgstr "Informieren - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:221
-msgid "contact.header-title"
-msgstr "Kontakt - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:228
-msgid "imprint.header-title"
-msgstr "Impressum - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:235
-msgid "barrierefreiheit.header-title"
-msgstr "Barrierefreiheit - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:242
-msgid "data_privacy.header-title"
-msgstr "Datenschutz - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:249
-msgid "agb.header-title"
-msgstr "Nutzungsbedingungen - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:261
-msgid "about.header-title"
-msgstr "Projektinfos - Der Steuerlotse für Rente und Pension"
-
-msgid "about_digitalservice.header-title"
-msgstr "Digital Service - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:287
-msgid "erica-error.header-title"
-msgstr "Übermittlungsfehler - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:291 app/routes.py:299
-msgid "400.header-title"
-msgstr "Fehler 400 - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:303
-msgid "429.header-title"
-msgstr "Fehler 429 - Der Steuerlotse für Rente und Pension"
-
-#: app/routes.py:199 app/routes.py:307
```

Comment:
I am not sure if all of that should be changed. Have you already merged main and ran babel again?

Reply:
Sry got many merge conflicts...